bgp/peers.go

```diff
@@ -101,7 +101,7 @@ func (s *Status) fetchPeeringStateFromPod(ctx context.Context, pod *corev1.Pod)
 	cmd := []string{"cilium", "bgp", "peers", "-o", "json"}
 	output, err := s.client.ExecInPod(ctx, pod.Namespace, pod.Name, defaults.AgentContainerName, cmd)
 	if err != nil {
-		return nil, fmt.Errorf("failed to fetch bgp state from %s: %v", pod.Name, err)
+		return nil, fmt.Errorf("failed to fetch bgp state from %s: %v - requires cilium >= v0.14.0", pod.Name, err)
```

Comment:
The supported Cilium version should be 1.13.2 : https://github.com/cilium/cilium/releases/tag/v1.13.2  

In my view, this is not the right thing to do. Consider few options
- Error can happen for any number of reasons - running an unsupported Cilium version is one of the possible reasons.
- Encoding version in code can go out of sync since CLI can be backported in future to older releases. 

In my opinion, the minimum cilium version required can be added in the help section of `cilium bgp peer`.